news/349.added

```diff
@@ -0,0 +1 @@
+Add @survey-publication-count restapi service. This returns the data from @@survey-publication-count as JSON.
```

Comment:
The issue number should be the one from the scrum repo https://github.com/euphorie/osha.oira/blob/248f3f40023e064897b63606bcf3978bce0ea194/pyproject.toml#L21